dll/3rdparty/libxslt/CMakeLists.txt

```diff
@@ -50,6 +46,5 @@ endif()
 add_pch(libxslt precomp.h SOURCE)
 add_cd_file(TARGET libxslt DESTINATION reactos/system32 FOR all)
 
-if(USE_CLANG_CL OR (NOT MSVC))
-    target_compile_options(libxslt PRIVATE -Wno-pointer-sign -Wno-unused-function)
-endif()
+target_compile_options(libxslt PRIVATE
+    $<$<NOT:$<C_COMPILER_ID:MSVC>>:-Wno-misleading-indentation;-Wno-pointer-sign;-Wno-unused-function>)
```

Comment:
Same thing here as above. Please keep the if syntax.

Reply:
Done